src/CoreBundle/Bridge/Chameleon/Module/NavigationTree/NavigationTree.php

```diff
@@ -572,7 +572,12 @@ private function createRootTree(): array
             $defaultPortal = $this->portalDomainService->getDefaultPortal();
             $defaultPortalMainNodeId = $defaultPortal->fieldMainNodeTree;
 
-            $portalList = \TdbCmsPortalList::GetList();
+            $portalsInMainTreeNodeOrderQuery = '
+                SELECT `cms_portal`.* FROM `cms_portal` 
+                    LEFT JOIN `cms_tree` ON `cms_portal`.`main_node_tree` = `cms_tree`.`id` 
```

Comment:
What does this filter out?

Reply:
Nothing (just a left join, not an inner join). It just provides the cms_tree fields needed in ORDER BY.